aiosmtpd/testing/helpers.py

```diff
@@ -65,19 +65,25 @@ def start(plugin):
         warnings.filterwarnings('always', category=ResourceWarning)
 
 
-def assert_auth_success(testcase: TestCase, code, response):
-    testcase.assertEqual(code, 235)
-    testcase.assertEqual(response, b"2.7.0 Authentication successful")
+def assert_auth_success(testcase: TestCase, *response):
+    testcase.assertEqual(
+        (235, b"2.7.0 Authentication successful"),
+        response
+    )
```

Comment:
Compare tuple against tuple directly so helpful error messages won't get hidden.

Also for `assert_auth_invalid` and `assert_auth_required` below.